internal/strace/strace.go

```diff
@@ -40,7 +40,8 @@ var (
 	unlinkPatten = regexp.MustCompile(`0x[a-f\d]+ ([^)]+)`)
 
 	// unlinkat(0x4 /tmp/pip-pip-egg-info-ng4_5gp_/temps.egg-info, 0x7fe0031c9a10 top_level.txt, 0x0)
-	unlinkatPattern = regexp.MustCompile(`0x[a-f\d]+ ([^,]+), 0x[a-f\d]+ ([^,]+), 0x[a-f\d]+`)
+	// unlinkat(AT_FDCWD /app, 0x5569a7e83380 /app/vendor/composer/e06632ca, 0x200)
+	unlinkatPattern = regexp.MustCompile(`\S+ ([^,]+), 0x[a-f\d]+ ([^,]+), 0x[a-f\d]+`)
```

Comment:
nit: can you use `[^\s]+` instead of `\S+` for consistency with the other regexes in this file?